wallet-new/integration/AccountSpecs.hs

```diff
@@ -98,6 +98,67 @@ accountSpecs wRef wc =
 
             map (AccountBalance . accAmount) accsUpdated `shouldBe` balancesPartialUpdated
 
+
+
+        it "redeeming avvv key gives rise to the corresponding increase of balance of wallet'account - mnemonic not used" $ do
+
+            newWallet <- randomWallet CreateWallet
+            Wallet{..} <- createWalletCheck wc newWallet
+
+            --adding new account
+            rAcc <- generate arbitrary :: IO NewAccount
+            newAcctResp <- postAccount wc walId rAcc
+            newAcct <- wrData <$> newAcctResp `shouldPrism` _Right
+
+            balancePartialRespB <- getAccountBalance wc walId (accIndex newAcct)
+            balancesPartialB <- wrData <$> balancePartialRespB `shouldPrism` _Right
+            let zeroBalance = AccountBalance $ V1 (Core.mkCoin 0)
+            balancesPartialB `shouldBe` zeroBalance
+
+            -- state-demo/genesis-keys/keys-fakeavvm/fake-9.seed
+            let avvmKey = "QBYOctbb6fJT/dBDLwg4je+SAvEzEhRxA7wpLdEFhnY="
+            --password is set to Nothing
+            passPhrase <- generate (pure mempty) :: IO SpendingPassword
+            let redemption = Redemption
+                    { redemptionRedemptionCode = ShieldedRedemptionCode avvmKey
+                    , redemptionMnemonic = Nothing
+                    , redemptionSpendingPassword = case newwalSpendingPassword newWallet of
+                            Just spPassw -> spPassw
+                            Nothing      -> passPhrase
+                    , redemptionWalletId = walId
+                    , redemptionAccountIndex = accIndex newAcct
+                    }
+
+            etxn <- redeemAda wc redemption
+
+            txn <- fmap wrData etxn `shouldPrism` _Right
+
+            threadDelay 180000000
```

Comment:
I would suggest to reduce the number of seconds you wait here -- this is inflating the CI running time of 3 minutes 😉 

What about ~30/60 seconds? That should be plenty for the node to accept the block with the new transaction.